server/config/subject_page.proto

```diff
@@ -164,9 +164,36 @@ message ScatterTileSpec {
 }
 
 message BarTileSpec {
+  enum SortType {
+    SORT_ASCENDING = 1;
+    SORT_DESCENDING = 2;
+    SORT_ASCENDING_POPULATION = 3;
+    SORT_DESCENDING_POPULATION = 4;
+  }
   // [optional] The root url to use for labels in the x-axis. If not specified,
   // labels in the x-axis will link to Place Explorer.
   string x_label_link_root = 1;
+  // Bar height for horizontal bar charts
+  double bar_height = 2;
+  // list of specific colors to use
+  repeated string colors = 3;
+  // whether to plot bars horizontally.
+  bool horizontal = 4;
+  // maximum number of places to display
+  int32 max_places = 5;
+  // bar sort order
+  // options:
+  // - "ascending"
+  // - "descending"
```

Comment:
The enums are self-explanatory?

SortType sort = 6 [default = SORT_DESCENDING_POPULATION]

Reply:
Removed the options comments. It looks like setting explicit defaults are disallowed in proto3, so I left a comment that default = descending population instead.